scripts/commit-msg.sh

```
@@ -0,0 +1,28 @@
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+
+# Run commitlint and capture the exit code
+npx --no -- commitlint --edit "$1"
+EXIT_CODE=$?
+
+# If commitlint failed, print a helpful guide
+if [ $EXIT_CODE -ne 0 ]; then
+  echo ""
+  echo -e "${RED}⛔️ Commit message format error!${NC}"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)